packages/fxa-content-server/app/tests/spec/views/base.js

```diff
@@ -180,6 +180,11 @@ describe('views/base', function() {
   });
 
   describe('render', function() {
+    it('renders react component correctly', () => {
+      if (view.renderReactComponent) {
```

Comment:
What needs to happen here is to create a method on the view called `renderReactComponent`, call `render()`, and then ensure the method is called. Right now no `renderReactComponent` method is on the view, so the test on line 185 isn't happening.

Have a look around at the uses of `sinon.spy`, that should give you the functionality that you need.